### PR TITLE
[AUTO-530] use native JS instead of lodash

### PR DESCRIPTION
**Pull request by thomaschaplin**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,9 +32,6 @@
     "dependencies": {
         "graphql": "^14.7.0",
         "json-to-graphql-query": "^1.9.0",
-        "lodash.isarray": "^4.0.0",
-        "lodash.isobject": "^3.0.2",
-        "lodash.isstring": "^4.0.1",
         "lodash.mapvalues": "^4.6.0"
```
Comment (thomaschaplin) [inline package.json:35]: Lodash mapvalues is tricker to replace with native JS. We should look to replace this in the future

**Review by rickschubert — APPROVED** [review]
I wasn't aware of `instanceof` - nice.

**Comment by thomaschaplin** [comment]
@rickschubert `you might not need lodash` 😉 - https://youmightnotneed.com/lodash/

File: src/index.ts [hunk 2]
```diff
@@ -74,6 +71,10 @@ interface ActualDefinitionNode {
 const undefinedVariableConst = "undefined_variable"
 const isVariableDropinConst = "_____isVariableDropinConst"
 
+const isString = (arg: any): boolean => typeof arg === "string"
+const isArray = Array.isArray
+const isObject = (arg: any): boolean => arg instanceof Object
```
Comment (rickschubert) [inline src/index.ts:76]: Do you know if `instanceof` also works in older nodejs versions @thomaschaplin ?

Comment (rickschubert) [reply src/index.ts:76]: Ah, seems like this exists since node 0.10.0 - perfect :)

**Review by benifietray — APPROVED** [review]

**Pull request merged**